crates/router/src/core/payments.rs

```diff
@@ -1885,6 +1885,14 @@ where
     dyn api::Connector:
         services::api::ConnectorIntegration<F, Req, router_types::PaymentsResponseData>,
 {
+    if !is_operation_complete_authorize(&operation)
```

Comment:
why are we checking for the operation here? 
can you use https://github.com/juspay/hyperswitch/blob/df7cf96c96d7a4a0d9245cba6e5b686ab6b6e5ea/crates/router/src/core/payments/flows/authorize_flow.rs#L268

Reply:
`fn preprocessing_steps` is overridden for Both `Authorize` and `CompleteAuthorize` flows.
But I want to call `preprocessing_steps` only during Authorize flow for Airwallex. 

Within `fn preprocessing_steps` implementation of Authorize Flow, we are calling `fn authorize_preprocessing_steps`

https://github.com/juspay/hyperswitch/blob/df7cf96c96d7a4a0d9245cba6e5b686ab6b6e5ea/crates/router/src/core/payments/flows/authorize_flow.rs#L125